src/select.coffee

```diff
@@ -213,6 +213,10 @@ Ember.AddeparMixins.ResizeHandlerMixin,
 
   contentProxy: Ember.computed ->
     optionLabelPath = @get('optionLabelPath')
+    if optionLabelPath is ''
```

Comment:
I don't think it's possible for `optionLabelPath` to be null or undefined. Doesn't the component have a default?